init.c

```diff
@@ -1576,7 +1576,12 @@ static int add_job(struct thread_data *td, const char *jobname, int job_add_num,
 	td->ts.sig_figs = o->sig_figs;
 
 	init_thread_stat_min_vals(&td->ts);
-	td->ddir_seq_nr = o->ddir_seq_nr;
+
+	//td->>ddir_seq_nr needs to be initialized to 1, NOT o->ddir_seq_nr,
```

Comment:
Please use proper C comments. For a multi-line comment, the fio style is:

```
/*
 * Here's a line
 * Here's another
*/
```

Reply:
Oops, sorry about that. Updated the PR.